yt_dlp/extractor/zaiko.py

```diff
@@ -90,3 +96,23 @@ def _real_extract(self, url):
                 'thumbnail': ('poster_url', {url_or_none}),
             })),
         }
+
+
+class ZaikoETicketIE(ZaikoBaseIE):
+    _VALID_URL = r'https?://(?:www.)?zaiko\.io/account/eticket/(?P<id>\w{48}=)'
```

Comment:
https://github.com/yt-dlp/yt-dlp/pull/7347#discussion_r1234217153
```suggestion
    _VALID_URL = r'https?://(?:www.)?zaiko\.io/account/eticket/(?P<id>[\w=-]{48}=)'
```